eth2/utils/bls/src/aggregate_signature.rs

```diff
@@ -112,6 +113,20 @@ impl AggregateSignature {
         Ok(Self::empty_signature())
     }
 
+    /// Returns the underlying signature.
+    pub fn as_raw(&self) -> &RawAggregateSignature {
+        &self.aggregate_signature
+    }
+
+    /// Returns the underlying signature.
+    pub fn from_point(point: G2Point) -> Self {
+        Self {
+            aggregate_signature: RawAggregateSignature { point },
+            // FIXME: check to ensure it is empty.
```

Comment:
Did you want to fix this now or later?

Reply:
Oh yeah good find! I just checked, this should _not_ be empty as it's created from a valid, potentially useful point.